style.css

```diff
@@ -0,0 +1,138 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  margin-top: 10px;
+}
+
+body {
+  position: absolute;
+  width: 375px;
+  height: 718px;
+  left: 0;
+  top: 0;
+  background: #fff;
+}
```

Comment:
- [ ] **[OPTIONAL]:** I see that you use the `position: absolute` property on multiple elements across your stylesheet. I strongly suggest you avoid using this property if you don't need to, as it creates a separate document flow for your elements and will make them hard to lay out. You could read more about this [here](https://ishadeed.com/article/less-absolute-positioning-modern-css/).